packages/assets-controllers/src/NftController.ts

```diff
@@ -512,6 +512,53 @@ export class NftController extends BaseController<
     });
   }
 
+  /**
+   * Fetches NFT Collection Metadata from the NFT API.
+   *
+   * @param contractAddresses - The contract addresses of the NFTs.
+   * @param chainId - The chain ID of the network where the NFT is located.
+   * @returns NFT collections metadata.
+   */
+  async fetchNftCollectionsMetadata(
+    contractAddresses: string[],
+    chainId: Hex,
+  ): Promise<{
+    collections: Collection[];
+  }> {
+    return await this.#getNftCollectionInformationFromApi(
+      contractAddresses,
+      chainId,
+    );
+  }
+
+  async #getNftCollectionInformationFromApi(
+    contractAddress: string[],
+    chainId: Hex,
+  ) {
+    const urlParams = new URLSearchParams({
+      chainId,
+    });
+
+    for (const address of contractAddress) {
+      urlParams.append('contract', address);
+    }
+
+    return await handleFetch(
+      `${this.#getNftCollectionApi()}?${urlParams.toString()}`,
+      {
+        headers: {
+          Version: '1',
```

Comment:
Nit; can use NFT_API_VERSION for the version

Reply:
Done in https://github.com/MetaMask/core/pull/4506/commits/1a8b4353a55e8198f51038b08eb5ee6c72147aa1